tests/conftest.py

```diff
@@ -397,3 +409,79 @@ def sentinel_stac_collection():
     return pystac.collection.Collection.from_file(
         str(TEST_DATA_FOLDER.joinpath(SENTINEL_STAC_COLLECTION))
     )
+
+
+@pytest.fixture
+def s2_l2a_stac():
```

Comment:
Can these fixtures get return types? (I think this one returns `pystac.item.Item`)